packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -19,43 +20,64 @@ export interface ITimePickerStrings {
 }
 
 export interface ITimePickerProps
-  extends Omit<IComboBoxProps, 'options' | 'selectedKey' | 'defaultSelectedKey' | 'multiSelect' | 'text'> {
+  extends Omit<
+    IComboBoxProps,
+    'options' | 'selectedKey' | 'defaultSelectedKey' | 'multiSelect' | 'text' | 'defaultValue'
+  > {
   /**
    * Label of the component
    */
   label?: string;
+
   /**
    * Time increments, in minutes, of the options in the dropdown
    */
   increments?: number;
+
   /**
    * If true, show seconds in the dropdown options and consider seconds for
    * default validation purposes.
    */
   showSeconds?: boolean;
+
   /**
    * If true, use 12-hour time format. Otherwise, use 24-hour format.
    */
   useHour12?: boolean;
+
   /**
    * If true, the TimePicker allows freeform user input, rather than restricting
    * to the default increments.
    *
    * The input will still be restricted to valid time values.
    */
   allowFreeform?: boolean;
+
   /**
    * Custom time range to for time options
    */
   timeRange?: ITimeRange;
+
   /**
    * Localized strings to use in the TimePicker
    */
   strings?: ITimePickerStrings;
+
+  /**
+   * Default value of the TimePicker, if any
+   */
+  defaultValue?: Date;
+
+  /**
+   * Callback issued when a time is selected
+   */
+  onSelectTime?: (time: Date) => void;
```

Comment:
This should have the event that triggered the change as the first parameter. That's the standard signature we try to use for callbacks (even if it's very inconsistent in legacy code).

Ideally it would be preferable to omit `onChange` from the inherited ComboBox props and rename this callback to `onChange`, but that would technically be a breaking change, and it's been long enough since the component was checked in that we probably shouldn't do that.

Reply:
Thank you for this suggestion! I went ahead and omitted `onChange` from the inherited ComboBox props and renamed `onSelectTime` to `onChange`. This approach feels cleaner and less confusing, great stuff.